coverart_redirect/request.py

```diff
@@ -244,6 +244,10 @@ def handle_robots(self):
         '''Serve up a permissive robots.txt'''
         return Response(response="User-agent: *\nAllow: /", mimetype='text/plain')
 
+    def handle_api(self, request):
+        '''Redirect to API docs at musicbrainz.org'''
+        return request.redirect (code=301, location='https://musicbrainz.org/doc/Cover_Art_Archive/API')
```

Comment:
There should be no space before `(`. See [PEP-8](https://www.python.org/dev/peps/pep-0008/).